deployments/cognito-rds-s3/terraform/main.tf

```diff
@@ -189,6 +239,7 @@ module "kubeflow_components" {
     aws = aws
     aws.virginia = aws.virginia
   }
+
```

Comment:
nit: is this coming from terraform fmt?, can we have a PR with all fmt changes separately for all tf files? ok for now, also add a GH action to check formatting in that PR

Reply:
was manually added, sure